File: IMC.py
#PEDIMOS DATOS PERSONALES
print("Ingresa tu primer nombre")
nombre = input()

edad = int(input(f"Hola {nombre}, ingresa tu edad: "))

print("\nAhora ingresa tu peso en kilogramos: ", end="")
peso = float(input())

print(f"\nMuy bien, {nombre}, ahora ingresa tu altura en centimetros: ", end="")
altura = float(input())

altura_metros = altura / 100
imc = peso / (altura_metros ** 2)

print("=" * 30)
print("         DIAGNOSTICO          ")
print("=" * 30)
print(f"Nombre: {nombre}\nEdad: {edad} años\nPeso: {peso:.2f} Kg.\nAltura: {altura_metros:.2f} m\n") #DELIMITAMOS A 2 DECIMALES CON .2f

print("RESULTADOS")
print(f"IMC calculado: {imc:.2f}")
#HACER CALCULOS DEL IMC
if imc <= 18.5:
    print("\nBAJO PESO\n")
elif 18.6 <= imc <= 24.9:
    print("\nPESO NORMAL\n")
elif 25.0 <= imc <= 29.9:
    print("\nSOBREPESO\n")
elif 30.0 <= imc <= 34.9:
    print("\nOBESIDAD TIPO 1\n")
elif 35.0 <= imc <= 39.9:
    print("\nOBESIDAD TIPO 2\n")
elif imc >= 40.0:
    print("\nOBESIDAD TIPO 3\n")
else:
    print("DATOS INVALIDOS\n")

File: IMC.c
#include <stdio.h>
char nombre[20];
int edad;
float peso, alturametros, altura, IMC;
int main()
{
    //PEDIMOS DATOS PERSONALES
    printf("Ingresa tu primer nombre\n");
    scanf("%s", nombre);
    printf("Hola %s, ingresa tu edad: ", nombre);
    scanf("%d", &edad);
    printf("\nAhora ingresa tu peso en kilogramos: ");
    scanf("%f", &peso);
    printf("\nMuy bien, %s, ahora ingresa tu altura en centimetros: ", nombre);
    scanf("%f", &altura);
    
    alturametros = altura / 100;
    IMC = peso / (alturametros * alturametros);
    printf("==============================\n");
    printf("         DIAGNOSTICO          \n");
    printf("==============================\n");
    printf("Nombre: %s\nEdad: %d\nPeso: %.2f\nAltura: %.2f\n\n", nombre, edad, peso, alturametros);//DELIMITAMOS A 2 DECIMALES CON .2f
    //HACER CALCULOS DEL IMC
    if (IMC <= 18.5){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nBAJO PESO\n\n");
    }
    else if (IMC >= 18.6 && IMC <= 24.9){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nPESO NORMAL\n\n");
    }
    else if (IMC >= 25.0 && IMC <= 29.9){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nSOBREPESO\n\n");
    }
    else if (IMC >= 30.0 && IMC <= 34.9){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nOBESIDAD TIPO 1\n\n");
    }
    else if (IMC >= 35.0 && IMC <= 39.9){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nOBESIDAD TIPO 2\n\n");
    }
    else if (IMC >= 40.0){
        printf("RESULTADOS\n");
        printf("IMC calculado: %.2f", IMC);
        printf("\nOBESIDAD TIPO 3\n\n");
    }
    else{
        printf("DATOS INVALIDOS\n\n");
    }
    return 0;
}